content/en/apps/tutorials/application-tests.md

```diff
@@ -0,0 +1,600 @@
+      "contact_type": "chw_area"
+    },
+    {
+      "_id": "patient_id",
+      "type": "contact",
+      "contact_type": "patient",
+      "name": "Patient Name",
+      "role": "patient",
+      "date_of_birth": "1970-07-09",
+      "sex": "female",
+      "reported_date": 1652868491000,
+      "patient_id": "patient_id",
+      "parent": {
+        "_id": "patient_parent_id",
+        "parent": {
+          "_id": "chw_area_id",
+          "parent": {
+            "_id": "district_id"
+          }
+        }
+      }
+    }
+  ]
+}
+{{< /highlight >}}
+
+Note that the `reported_date` above stores the epoch timestamp in milliseconds when the document was first created. You can use external website [epochconverter](https://www.epochconverter.com/) to convert the timestamp to and from a human readable date.
+
+After setting the harness defaults, you can now test the task:
+```js {linenos=true}
+// Get the task by title
+const tasks = await harness.getTasks({title: 'First Assessment'});
+
+//Make sure that you have the task
+expect(tasks).to.have.property('length', 1);
+
+// Load the task action and fill it.
+const result = await harness.loadAction(tasks[0], ['yes'], ['7']);
+
+// Verify that the form successfully got submitted
+expect(result.errors).to.be.empty;
+
+// Verify that the task is no longer visible
+expect(await harness.getTasks()).to.be.empty;
+```
+
+If a task is triggered by a report, then fill in the app form to create report before test:
+
+```js {linenos=table}
+it('followup schedule', async () => {
+  const result = await harness.fillForm(...pncFollowups.registerBirth('2000-01-01', '2000-02-01'));
+  expect(result.errors).to.be.empty;
+
+  /* Task should appear 7 days after birth for follow up */
+  await harness.flush(7);
+  const tasks = await harness.getTasks();
+  expect(tasks).to.have.property('length', 1);
+   
+  /* Complete the task, task should disappear */
+  await harness.loadAction(tasks[0].emission.actions[0]);
+  const followupResult = await harness.fillForm(...pncFollowups.followup.healthy);
+  expect(followupResult.errors).to.be.empty;
+  expect(await harness.getTasks()).to.be.empty;
+
+  /* Confirm a task appears again on the schedule */
+  await harness.flush(21);
+  expect(await harness.getTasks()).to.have.property('length', 1);
+});
+```
+
+You may [pass other data](https://docs.communityhealthtoolkit.org/apps/guides/tasks/pass-data-to-form/) from a task to the action form using [modifyContent ](https://docs.communityhealthtoolkit.org/apps/guides/tasks/pass-data-to-form/#modifycontent)attribute of a task. You can also verify that these data are present in the task.
+
+
+```js
+    expect(tasks[0].emission.actions[0].content).to.include({
+      t_follow_up_count: '5',
+      t_delivery_date:'2000-01-01',
+    });
+
+```
+--- 
+### Testing Targets
+
+Testing a target is relatively straightforward. Add a report or contact that increments a target, then check the target values.
+
+<table>
```

Comment:
Similar to above comments on table, please refer to: https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#use-markdown-notation

Reply:
Converted. Thanks!